src/layout/node.ts

```diff
@@ -3,6 +3,8 @@ import config from "@src/core/config";
 import * as dom from "./dom";
 
 const IGNORE_ATTRIBUTES = ["title", "alt", "onload", "onfocus"];
+const SVG_NAMESPACE: string = "http://www.w3.org/2000/svg";
+const SVG_PREFIX: string = "svg:";
```

Comment:
You repeat this in the dom.ts file. Remove one of them.

Reply:
Thanks, done. Changed all const strings into const enum and now we don't have to waste bytes sharing the variable name across files - which don't get minified later.